src/bitmap.c

```diff
@@ -0,0 +1,189 @@
+/*	bitmap.c
+ *
+ *	Source: https://msdn.microsoft.com/de-de/library/windows/desktop/dd145119(v=vs.85).aspx
+ *
+ *	Usage:
+ *		HBITMAP hBitmap;
+ *		HDC hdc;
+ *		PBITMAPINFO pBitmapInfo = createBitmapInfoStruct(hBitmap);
+ *		createBmpFile("picture.bmp"), pBitmapInfo, hBitmap, hdc);
+ */
+
+#if defined(WIN32_LEAN_AND_MEAN)
```

Comment:
MSYS2 and Cygwin GCC don't define `_WIN32`. Should I replace it with `#if defined(_WIN32) || defined(__CYGWIN__) || defined(__MSYS__)` instead?